packages/mobile/src/utils/numberFormat.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
```

Comment:
consider merging this file with utils/formatting.ts or renaming it to make it clear its for the HOC